lib/src/searchfield.dart

```diff
@@ -7,6 +7,17 @@ enum SuggestionState {
 
   /// hide the suggestions on initial focus
   hidden,
+
+  /// show suggestions only on tap
+  onTap,
+}
+
+enum SuggestionAction {
+  /// shift to next focus
+  shift,
```

Comment:
this enum should be renamed to `next` instead of `shift` to maintain consistency with `SearchInputAction`.